server/controllers/auth.js

```diff
@@ -37,6 +48,9 @@ exports.registerUser = asyncHandler(async (req, res, next) => {
       maxAge: secondsInWeek * 1000
     });
 
+    // add user to online users list
+    userJoin(user);
```

Comment:
Feels a little weird to do this on the backend, would make more sense to handle this on the frontend socket client